src/main/resources/META-INF/plexus/components.xml

```diff
@@ -17,6 +22,7 @@
        <package>org.apache.maven.plugins:maven-jar-plugin:2.6:jar,${project.groupId}:${project.artifactId}:${project.version}:build</package>
        <pre-integration-test>${project.groupId}:${project.artifactId}:${project.version}:start</pre-integration-test>
        <integration-test>org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:integration-test</integration-test>
+       <!-- post-integration-test is opportunity for code coverage, such as jacoco, to collect statistics from running container -->
        <post-integration-test></post-integration-test>
        <verify>${project.groupId}:${project.artifactId}:${project.version}:stop,org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:verify</verify>
```

Comment:
Even after moving `docker:stop` to _post-integration-test_, calling `mvn integration-test` won't run _post-integration-test_ and therefore not stop the container.

Unfortunately, mounting the directory is a non-starter in our build chain. Let me look at preStop tonight.